frontend/app/src/entities/schema/ui/attribute-display.tsx

```diff
@@ -101,5 +104,26 @@ const AttributeParameters = ({
     );
   }
 
+  if (attribute.kind === "NumberPool") {
+    return (
+      <div>
+        <PropertyTitle title="Parameters" />
+
+        <div className="pl-4">
+          <PropertyRow
+            title="Number pool"
+            value={
+              <Link to={constructPath(`/resource-manager/${attribute.parameters?.number_pool_id}`)}>
+                <NodeLabel id={attribute.parameters?.number_pool_id} />
+              </Link>
+            }
+          />
+          <PropertyRow title="Start range" value={attribute.parameters?.start_range} />
+          <PropertyRow title="End range" value={attribute.parameters?.end_range} />
```

Comment:
it could be nicer for ou users if you format the number to make it easier to read